go.mod

```diff
@@ -8,66 +8,63 @@ require (
 	github.com/evanphx/json-patch v4.2.0+incompatible
 	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
 	github.com/golang/mock v1.2.0
-	github.com/golang/protobuf v1.3.1
+	github.com/golang/protobuf v1.3.2
 	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.6.1
 	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
-	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8 // indirect
-	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45 // indirect
-	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
-	google.golang.org/genproto v0.0.0-20190418145605-e7d98fc518a7 // indirect
 	google.golang.org/grpc v1.19.1
-	gopkg.in/inf.v0 v0.9.1 // indirect
-	k8s.io/api v0.0.0-20191005115622-2e41325d9e4b
-	k8s.io/apimachinery v0.0.0-20191006235458-f9f2f3f8ab02
-	k8s.io/client-go v0.0.0-20190918200256-06eb1244587a
-	k8s.io/csi-translation-lib v0.0.0-20191009030015-17db17aaadeb
+	k8s.io/api v0.0.0-20191122220107-b5267f2975e0
+	k8s.io/apimachinery v0.0.0-20191121175448-79c2a76c473a
+	k8s.io/client-go v0.0.0-20191122220542-ed16ecbdf3a0
+	k8s.io/csi-translation-lib v0.0.0-20191122224157-518651485193
 	k8s.io/klog v1.0.0
 )
 
-replace k8s.io/api => k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
+replace k8s.io/api => k8s.io/api v0.0.0-20191122220107-b5267f2975e0
```

Comment:
Which version of Kubernetes is that? Can you record that in the commit message?

I assume all code coming from Kubernetes is at that version?

Reply:
I vaguely remember seeing a script that can update all k8s dependencies. Was that something you wrote?

Reply:
I took some snippet that was posted in one of the Kubernetes discussions around semantic versioning and added it to csi-release-tools, so it's now in all of our repos: https://github.com/kubernetes-csi/csi-release-tools/blob/master/go-get-kubernetes.sh

But it's impossible to tell from the result whether that script was used and thus whether the new replacements are consistent. We basically have to trust the PR submitter that they did the right thing. To reproduce it, the version given to the scripts needs to be documented, hence me question about the commit message.

Reply:
> Which version of Kubernetes is that? Can you record that in the commit message?

It was `v1.17.0-rc.1`, but today I updated it to `v1.17.0-rc.2` (which was released yesterday). I also updated the commit message to record both the k8s version and how the dependencies were updated (using the `go-get-kubernetes.sh` script).